Assets/UI/Popups/greatpeoplepopup.xml

```diff
@@ -3,9 +3,9 @@
 
```

Comment:
In this file I would change the `hide="1"` to `hide="0"` for the Portrait (Image ID="Portrait") to show the new icon, and change its size to `120, 120`

Reply:
Done.
I totally do not know why, but I think you know the code. :D

Reply:
CQUI modified this portion of the code to not show the portrait, but now that there're new icons, we're going to use them so we don't hide anymore (hide=0). Also the default size if too big, CQUI condensed the screen to avoid any scrolling.